great_expectations/checkpoint/actions.py

```diff
@@ -125,63 +121,6 @@ def _using_cloud_context(self) -> bool:
 
         return project_manager.is_using_cloud()
 
-    @public_api
-    def run(
```

Comment:
All `run` and `_run` methods can be deleted on actions - we will promote `v1_run` to `run` in a follow-up